src/ui/CategoryFacet/CategoryFacetSearch.ts

```diff
@@ -108,7 +108,9 @@ export class CategoryFacetSearch implements IFacetSearch {
   }
 
   private handleClickElsewhere(e: MouseEvent) {
-    if (!$$(<HTMLElement>e.target).closest('.coveo-category-facet-search-container')) {
+    const closestContainer = $$(<HTMLElement>e.target).closest('.coveo-category-facet-search-container');
+
+    if (!closestContainer || closestContainer != this.container.el) {
```

Comment:
Should we have a unit test to cover this situation?

Reply:
I always feel dirty when I have to add UT for what is essentially "mouse clicks".

Those are always super clunky.

Reply:
Ok